Jest 27 - Paths not being honored

## 🐛 Bug Report

<!-- A clear and concise description of what the bug is. -->
We have a project with an Angular library, and are using tsconfig paths to help with import paths for components in the library.  This worked great before the update, but is now failing with "Cannot find module" for all of the imports involving components from the library.

Snippet of our tsconfig
```
"paths": {
      "@fabric/components/*": [
        "./dist/components/*"
      ]
    }
```

## Error log:
<!-- You can copy your stacktrace and paste it in the below section: 
-->

```
# content of error stacktrace :
 FAIL  src/app/component-demos/component-demos.component.spec.ts
  ● Test suite failed to run

    Cannot find module '@fabric/components/column-configuration-dialog' from 'src/app/component-demos/component-demos.component.spec.ts'

      38 | import { MatTreeModule } from '@angular/material/tree';
      39 | import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
    > 40 | import { FabColumnConfigurationDialogModule } from '@fabric/components/column-configuration-dialog';
         | ^
      41 | import { FabConfirmationDialogModule } from '@fabric/components/confirmation-dialog';
      42 | import { FabGmLogoModule } from '@fabric/components/gm-logo';
      43 | import { FabMultiselectModule } from '@fabric/components/multiselect';

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/index.js:311:11)
      at Object.<anonymous> (src/app/component-demos/component-demos.component.spec.ts:40:1)

```

## envinfo

```
System:
    OS: Windows 10

Npm packages:
    jest: 27.0.1
    jest-preset-angular: 9.0.0
    typescript: 4.2.6
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -73,7 +73,18 @@ module.exports = {
 * When generating a new project from Angular CLI, by default the `tsconfig.json` doesn't contain any path mappings
   hence removing `moduleNameMapper` from preset will make sure that the preset works in pair with `tsconfig.json`.
   Ones who are relying on the value of `moduleNameMapper` from the preset should create their own `moduleNameMapper`
-  config manually or via `ts-jest` util https://kulshekhar.github.io/ts-jest/docs/getting-started/paths-mapping
+  config manually or via `ts-jest` util https://kulshekhar.github.io/ts-jest/docs/getting-started/paths-mapping. 
+
+If you wish to reuse the old configuration of `moduleNameMapper`, you can put this into your Jest config
+```
+moduleNameMapper: {
+  '^src/(.*)$': '<rootDir>/src/$1',
+  '^app/(.*)$': '<rootDir>/src/app/$1',
+  '^assets/(.*)$': '<rootDir>/src/assets/$1',
+  '^environments/(.*)$': '<rootDir>/src/environments/$1',
+}
+```
+
 * By default, if `skipLibCheck` is not defined in tsconfig, `jest-preset-angular` will set it to `true`. If one wants to have it as `false`, one can set explicitly in tsconfig.
 * **compiler:** `jest-preset-angular` now switches to default to use its own transformer which wraps around `ts-jest` to transform codes.
 

--- a/website/docs/getting-started/options.md
+++ b/website/docs/getting-started/options.md
@@ -8,6 +8,22 @@ in the `package.json` file of your project, or through a `jest.config.js`, or `j
 
 More information about `ts-jest` options, see https://kulshekhar.github.io/ts-jest/docs/next/getting-started/options
 
+:::important
+
+Since **9.0.0**, `jest-preset-angular` default Jest configuration no longer provides `moduleNameMapper`. If you wish to reuse
+the old `moduleNameMapper` configuration, you can put this into your Jest config
+
+```
+moduleNameMapper: {
+  '^src/(.*)$': '<rootDir>/src/$1',
+  '^app/(.*)$': '<rootDir>/src/app/$1',
+  '^assets/(.*)$': '<rootDir>/src/assets/$1',
+  '^environments/(.*)$': '<rootDir>/src/environments/$1',
+}
+```
+
+:::
+
 ### Exposed configuration
 
 ```js

--- a/website/versioned_docs/version-9.x/getting-started/options.md
+++ b/website/versioned_docs/version-9.x/getting-started/options.md
@@ -8,6 +8,22 @@ in the `package.json` file of your project, or through a `jest.config.js`, or `j
 
 More information about `ts-jest` options, see https://kulshekhar.github.io/ts-jest/docs/next/getting-started/options
 
+:::important
+
+Since **9.0.0**, `jest-preset-angular` default Jest configuration no longer provides `moduleNameMapper`. If you wish to reuse
+the old `moduleNameMapper` configuration, you can put this into your Jest config
+
+```
+moduleNameMapper: {
+  '^src/(.*)$': '<rootDir>/src/$1',
+  '^app/(.*)$': '<rootDir>/src/app/$1',
+  '^assets/(.*)$': '<rootDir>/src/assets/$1',
+  '^environments/(.*)$': '<rootDir>/src/environments/$1',
+}
+```
+
+:::
+
 ### Exposed configuration
 
 ```js

